INSTALL.md

```diff
@@ -14,16 +14,15 @@ The following external libraries are required when building libFMS
 
 We strive to have libFMS built with as many C and Fortran compilers on as many
 Unix/Linux type systems as possible.  However, internally, we only use libFMS
-compiled with the Intel compilers.  Some group have had success using libFMS
+compiled with the Intel compilers.  Some groups have had success using libFMS
 when compiled with the GNU C and Fortran compilers.
 
 ## MPI Support
 
 The default way to build libMPI is with MPI support.  We have found that using
```

Comment:
Should this be "libFMS with MPI support"?

Reply:
This is marked as resolved, but I still see "build libMPI with MPI support" in the text.

Reply:
My bad.  I thought I fixed it with the other update.  I'll have it updated soon.

Reply:
Update pushed.